tests/ui/use_self.rs

```diff
@@ -462,3 +462,13 @@ mod issue6818 {
         a: i32,
     }
 }
+
+mod issue7206 {
+    struct MyStruct<const C: char>;
+
+    impl From<MyStruct<'a'>> for MyStruct<'b'> {
+        fn from(_s: MyStruct<'a'>) -> Self {
```

Comment:
Wait, how doesn't this fail our test suite?

```suggestion
    impl From<MyStruct<'a>> for MyStruct<'b> {
        fn from(_s: MyStruct<'a>) -> Self {
```

Same in the other test file changes.

Reply:
Oh nevermind, those are const generics as `char`s. Those are valid Rust code now. I have to get used to them I guess. :sweat_smile: